src/components/pages/register.tsx

```diff
@@ -1,21 +1,79 @@
+import { useState } from 'react'
+
+import { Button } from '@/components/atoms/button.tsx'
 import { Input } from '@/components/atoms/input.tsx'
 import { Typography } from '@/components/atoms/typography.tsx'
 import { Footer } from '@/components/molecules/footer.tsx'
+import { useToast } from '@/hooks/use-toast.ts'
+import { useRegisterAccountMutation } from '@/lib/redux/features/account/api.ts'
+import { RegisterAccountRequestArgs, RegisterAccountResponse } from '@/lib/redux/features/account/validation.ts'
+
+interface RegisterError {
+  status: number
+  data: Partial<RegisterAccountResponse>
+}
```

Comment:
This could be replaced with `FetchBaseQueryError`

Reply:
However with Andrew's latest implementation this no longer needed

Reply:
Okay thanks fer